### PR TITLE
Converted to SCSS from LESS which closes #33

### LINKED ISSUE #33: move to sass
despite having just moved to less id like to move us to sass
if we did we could add materialize as  a dep and import its sass into the build process

### PR DESCRIPTION
**Pull request by clementallen**

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -0,0 +1,169 @@
+@import url(//fonts.googleapis.com/css?family=Roboto:100,300,400);
+@import 'variables';
+
```
Comment (butlerx) [inline css/main.scss]: could you import 
`npm install materialize-css`

File: gulpfile.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 var connect = require('gulp-connect');
 var jsonSchema = require('gulp-json-schema');
 
-gulp.task('dev', ['compress', 'minify-css', 'less', 'webserver', 'validate'], function () {
-    gulp.watch(['./css/*.less', './js/*.js', './**/*.html'], ['less', 'compress', 'html']);
+gulp.task('dev', ['compress', 'minify-css', 'scss', 'webserver', 'validate'], function () {
+    gulp.watch(['./css/*.scss', './js/*.js', './**/*.html'], ['scss', 'compress', 'html']);
 });
 
 gulp.task('compress', function() {
```
Comment (butlerx) [inline gulpfile.js]: wouldnt be needed for after import of materialize

File: css/main.scss (hunk 1 above)
Comment (clementallen) [inline css/main.scss]: Will do

File: gulpfile.js (hunk 2 above)
Comment (butlerx) [inline gulpfile.js]: oh this is the wrong line sorry i meant the minify-css task

File: css/main.scss (hunk 1 above)
Comment (clementallen) [inline css/main.scss]: @butlerx would you like the `materialize.js` file that npm installs to be concatenated into `main.min.js` as part of the gulp process?

Comment (butlerx) [inline css/main.scss]: yes please :)

File: gulpfile.js (hunk 2 above)
Comment (clementallen) [inline gulpfile.js]: @butlerx that makes more sense!

File: css/main.scss (hunk 1 above)
Comment (clementallen) [inline css/main.scss]: @butlerx done

**Comment by butlerx** [comment]
@clementallen could you also delete the old materialize css file

**Comment by clementallen** [comment]
@butlerx file removed

**Comment by butlerx** [comment]
https://github.com/redbrick/hexo-theme-icarus/issues/12
if you are interested in similar issue again

**Pull request merged**